File: exercicios lp/exercicios lp/ex1.c
#include <stdio.h>


int main(){


    float fahrenheit;
    float resp;
    
    printf("Digite a temperatura que deve ser convertida: ");
    scanf("%f",&fahrenheit);

    resp = ((fahrenheit-32)*5)/9;

    printf("A temperatura %f em graus Fahrenheit, sera %.2f graus Celsius", fahrenheit,resp);

    return(0);
}

File: exercicios lp/exercicios lp/go/ex1.go
package main

import "fmt"

func main() {

	var celsius, fahrenheit float64

	fmt.Print("Informe a temperatura a ser convertida: ")
	fmt.Scan(&fahrenheit)

	celsius = (((fahrenheit - 32) * 5) / 9)

	fmt.Printf("A temperatura convertida é: %f Graus Celsius", celsius)
}
